addon/session-stores/cookie.js

```diff
@@ -143,26 +167,22 @@ export default BaseStore.extend({
   },
 
   _read(name) {
-    let value = document.cookie.match(new RegExp(`${name}=([^;]+)`)) || [];
-    return decodeURIComponent(value[1] || '');
+    return this.get('_cookies').read(name) || '';
   },
 
   _calculateExpirationTime() {
-    let cachedExpirationTime = this._read(`${this.cookieName}-expiration_time`);
+    let cachedExpirationTime = this._read(`${this.cookieName}:expiration_time`);
```

Comment:
same as above - we shouldn't change the cookie name here unless absolutely necessary